lldb/test/API/lang/swift/typerefs/objc-descendent/TestSwiftObjCDescendantClassWithoutASTContext.py

```diff
@@ -0,0 +1,17 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+
+class TestCase(TestBase):
+    @swiftTest
+    @skipUnlessFoundation
+    def test(self):
+        """Print an ObjC derived object without using the AST context."""
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "// break here", lldb.SBFileSpec("main.swift")
+        )
+        self.runCmd("settings set symbols.swift-enable-ast-context false")
+        self.expect("v", substrs=["num = 15"])
```

Comment:
it would be nice to have a policy on this. The `v` alias could change, but it would have to be a breaking change for a test like this to be a problem. A breaking change seems unlikely.

Reply:
There are other instances, I'll make a follow up change to catch them all.

Reply:
https://github.com/swiftlang/llvm-project/pull/9351